nx-dev/feature-ai/src/lib/feature-ai.tsx

```diff
@@ -73,43 +86,43 @@ export function FeatureAi(): JSX.Element {
           <h1>Thinking...</h1>
         </div>
       ) : null}
-      {finalResult && !error ? (
+      {finalResult && !loading && !error ? (
         <>
           <div className="p-4 max-w-none prose prose-slate dark:prose-invert">
             {finalResult}
           </div>
-          <div>
-            <Button
-              variant="primary"
-              size="small"
-              onClick={() =>
-                sendCustomEvent('ai_feedback', 'ai', 'good', undefined, {
-                  query,
-                  result: finalResult,
-                })
-              }
-            >
-              Answer was helpful{' '}
-              <span role="img" aria-label="thumbs-up">
-                👍
-              </span>
-            </Button>
-            <Button
-              variant="primary"
-              size="small"
-              onClick={() =>
-                sendCustomEvent('ai_feedback', 'ai', 'bad', undefined, {
-                  query,
-                  result: finalResult,
-                })
-              }
-            >
-              Answer looks wrong{' '}
-              <span role="img" aria-label="thumbs-down">
-                👎
-              </span>
-            </Button>
-          </div>
+          {!feedbackSent && (
+            <div>
+              <Button
+                variant="primary"
+                size="small"
+                onClick={() => handleFeedback('good')}
+              >
+                Answer was helpful{' '}
+                <span role="img" aria-label="thumbs-up">
+                  👍
+                </span>
+              </Button>
+              <Button
+                variant="primary"
+                size="small"
+                onClick={() => handleFeedback('bad')}
+              >
+                Answer looks wrong{' '}
+                <span role="img" aria-label="thumbs-down">
+                  👎
+                </span>
+              </Button>
+            </div>
+          )}
+          {feedbackSent && (
+            <p>
+              <span role="img" aria-label="check">
+                ✅
+              </span>{' '}
+              Thank you for your feedback!
+            </p>
```

Comment:
I think we can still make this prettier but it's low priority and can be done in a follow up.